src/reducers/signup.reducer.js

```diff
@@ -0,0 +1,31 @@
+import SIGNUP from '../constants/index';
+import signupInitialState from './initialState';
```

Comment:
Once you've implemented the feedback on initial state, your import will look like this:
```import initialState from './initialState';```